README.md

```diff
@@ -35,6 +35,14 @@ timesync_ntp_servers:
                                 # of the hostname is a separate NTP server
                                 # (default no)
 
+# List of NTP peers
+timesync_ntp_peers:
```

Comment:
We would probably want to keep the API consistent with pool, so instead of adding a new variable for peers, there should just be a new flag in ntp_servers

Reply:
Refactored in 17a3a7d6c112c2a686edf59372168b0121595143